src/widgets/attachments/attachments-overview.component.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useState, useEffect } from "react";
+import { useCurrentPatient } from "@openmrs/esm-api";
+import {
+  getAttachments,
+  createAttachment,
+  deleteAttachment
+} from "./attachments.resource";
+import Gallery from "react-grid-gallery";
+import styles from "./attachments-overview.css";
+
+export default function AttachmentsOverview() {
+  const [attachments, setAttachments] = useState([]);
+  const [currentImage, setCurrentImage] = useState(0);
+
+  const [
+    isLoadingPatient,
+    patient,
+    patientUuid,
+    patientErr
+  ] = useCurrentPatient();
+
+  useEffect(() => {
+    if (patientUuid) {
+      const abortController = new AbortController();
+      getAttachments(patientUuid, true, abortController).then(
+        (response: any) => {
+          const listItems = response.data.results.map(attachment => ({
+            id: `${attachment.uuid}`,
+            src: `/openmrs/ws/rest/v1/attachment/${attachment.uuid}/bytes`,
```

Comment:
@ivange94 @FlorianRappl will this work when the context path is not `openmrs`?

Reply:
No it will not work and we should not build urls like this.